ansible.cfg

```diff
@@ -0,0 +1,19 @@
+[defaults]
```

Comment:
Do you really need a custom ansible.cfg to use the role? Why? Our roles should work with the default `ansible.cfg`

Reply:
Sorry, taking this out; just need it for some of our environments.